src/main/scala/cromwell/engine/workflow/WorkflowActor.scala

```diff
@@ -438,8 +446,8 @@ case class WorkflowActor(workflow: WorkflowDescriptor, backend: Backend)
     completionWork onFailure {
       case e =>
         logger.error(s"Completion work failed for call ${callKey.tag}! " + e.getMessage, e)
-        scheduleTransition(WorkflowFailed)
         self ! PersistenceFailed(callKey, ExecutionStatus.Done)
+        self ! CheckForWorkflowComplete
```

Comment:
Maybe. If there's something in pending, the CheckForCompletions won't trigger. 

So, I'll compose these two, instead of sending both and hoping for the best.